src/components/Calculator.js

```diff
@@ -1,62 +1,54 @@
-        <Screen content={next || total} />
-        <div className="row">
-          <GrayBtn symbol="AC" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="+/-" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="%" btnHandler={this.btnHandler} />
-          <OrangeBtn symbol="÷" btnHandler={this.btnHandler} />
-        </div>
-        <div className="row">
-          <GrayBtn symbol="7" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="8" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="9" btnHandler={this.btnHandler} />
-          <OrangeBtn symbol="x" btnHandler={this.btnHandler} />
-        </div>
-        <div className="row">
-          <GrayBtn symbol="4" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="5" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="6" btnHandler={this.btnHandler} />
-          <OrangeBtn symbol="-" btnHandler={this.btnHandler} />
-        </div>
-        <div className="row">
-          <GrayBtn symbol="1" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="2" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="3" btnHandler={this.btnHandler} />
-          <OrangeBtn symbol="+" btnHandler={this.btnHandler} />
-        </div>
-        <div className="row">
-          <BigGrayBtn symbol="0" btnHandler={this.btnHandler} />
-          <GrayBtn symbol="." btnHandler={this.btnHandler} />
-          <OrangeBtn symbol="=" btnHandler={this.btnHandler} />
-        </div>
+    const current = state;
+    setState(calculate(current, keyPressed));
+  };
+  const { total, next } = state;
+  return (
+    <div className="calc-body">
+      <Screen content={next || total} />
+      <div className="row">
+        <GrayBtn symbol="AC" btnHandler={btnHandler} />
+        <GrayBtn symbol="+/-" btnHandler={btnHandler} />
+        <GrayBtn symbol="%" btnHandler={btnHandler} />
+        <OrangeBtn symbol="÷" btnHandler={btnHandler} />
+      </div>
+      <div className="row">
+        <GrayBtn symbol="7" btnHandler={btnHandler} />
+        <GrayBtn symbol="8" btnHandler={btnHandler} />
+        <GrayBtn symbol="9" btnHandler={btnHandler} />
+        <OrangeBtn symbol="x" btnHandler={btnHandler} />
+      </div>
+      <div className="row">
+        <GrayBtn symbol="4" btnHandler={btnHandler} />
+        <GrayBtn symbol="5" btnHandler={btnHandler} />
+        <GrayBtn symbol="6" btnHandler={btnHandler} />
+        <OrangeBtn symbol="-" btnHandler={btnHandler} />
+      </div>
+      <div className="row">
+        <GrayBtn symbol="1" btnHandler={btnHandler} />
+        <GrayBtn symbol="2" btnHandler={btnHandler} />
+        <GrayBtn symbol="3" btnHandler={btnHandler} />
+        <OrangeBtn symbol="+" btnHandler={btnHandler} />
+      </div>
+      <div className="row">
+        <BigGrayBtn symbol="0" btnHandler={btnHandler} />
+        <GrayBtn symbol="." btnHandler={btnHandler} />
+        <OrangeBtn symbol="=" btnHandler={btnHandler} />
       </div>
```

Comment:
- [ ] [OPTIONAL] Good :+1: job refactoring your code with hooks but  I can see that you have repeatedly used `symbol`, I think storing your `symbol` into an array would make your code look more awesome, You may try something like this :point_down: 
![image](https://user-images.githubusercontent.com/75126481/143476421-6ccf7a9f-d372-46b8-8b93-5fa48f7c5ec5.png)